stacks/environment-aws/variables.tf

```diff
@@ -104,6 +104,14 @@ variable "enable_keycloak" {
   default = true
 }
 
+variable "enable_google_login" {
+  default = false
+}
+
+variable "enable_test_user" {
+  default = false
+}
+
```

Comment:
environment-local also?

Reply:
I can go ahead and add this to environment-local, but I have no way of testing it since environment-local is already really out of date, and I don't want the scope of this PR to include fixing that as well.